src/main/java/com/google/devtools/build/lib/actions/cache/CompactPersistentActionCache.java

```diff
@@ -497,6 +517,7 @@ private static byte[] encode(StringIndexer indexer, ActionCache.Entry entry) thr
     int maxOutputFilesSize =
         VarInt.MAX_VARINT_SIZE // entry.getOutputFiles().size()
             + (VarInt.MAX_VARINT_SIZE // execPath
+                    + (2 * VarInt.MAX_VARINT_SIZE) // symlinkTargetExecPath optional
```

Comment:
0 vs 1 you know is 1 byte, so it's more more like 1 + VarInt.MAX_VARINT_SIZE

Reply:
Done.